src/RokuDeploy.ts

```diff
@@ -581,6 +422,11 @@ export class RokuDeploy {
         let parsedManifest = await this.parseManifest(manifestPath);
         let appName = parsedManifest.title + '/' + parsedManifest.major_version + '.' + parsedManifest.minor_version;
 
+        //prevent devId mismatch (if devId is specified)
+        if (options.devId && options.devId !== await this.getDevId()) {
+            throw new Error('devId mismatch. nope, not gonna sign');
```

Comment:
Let's make a better error message here. Maybe something like:
```
`Package signing cancelled: devId '${options.devId}' does not match on-device devId `${onDeviceDevId}`
```

Reply:
Done and added test for it